### PR TITLE
enh(objectivec) add scope `types`

### PR DESCRIPTION
**Pull request by joshgoebel**
### Checklist
- [x] Added markup tests, or they don't apply here because we don't need markup tests for basic keywords support
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

**Comment by ElijahPepe** [comment]
LGTM.

File: src/languages/objectivec.js [hunk 1]
```diff
@@ -12,46 +12,51 @@ export default function(hljs) {
...
+    "signed",
+    "short",
+    "long",
+    "double",
+    "wchar_t",
+    "unichar",
+    "void",
+    "bool",
+    "BOOL",
+    "id|0",
```
Comment (allejo) [inline src/languages/objectivec.js:29]: What's this? I can't find any documentation on what `id|0` means in obj-c.

Comment (joshgoebel) [reply src/languages/objectivec.js:29]: <https://www.google.com/search?client=safari&rls=en&q=objective+c+types+%22id%22&ie=UTF-8&oe=UTF-8>

Just what google told me...

Comment (allejo) [reply src/languages/objectivec.js:29]: Ah I was looking for the `id|0` as a whole and couldn't find anything. What does the `|0` mean?

Comment (joshgoebel) [reply src/languages/objectivec.js:29]: 0 relevance.

Comment (allejo) [reply src/languages/objectivec.js:29]: 🤦  I didn't realize that was shorthand for relevance

**Review by allejo — APPROVED** [review]

**Pull request merged**